owl/SOMA-OBJ.owl

```diff
@@ -1906,9 +1939,19 @@ SubClassOf(SOMA:PathRole SOMA:SpatioTemporalRole)
 AnnotationAssertion(rdfs:comment SOMA:Patient "A role classifying an object that undergoes/is the primary object affected by the event.")
 SubClassOf(SOMA:Patient SOMA:EventAdjacentRole)
 
+
+# Class: SOMA:PerformerRole (Performer role)
+
+AnnotationAssertion(rdfs:comment SOMA:PerformerRole "A role classifying an Agent responsible for performing an Action.
```

Comment:
It seems this annotation needs an update to reflect the renaming AgentRole -> PerformerRole.